vpr/src/power/power_util.cpp

```diff
@@ -426,8 +421,9 @@ static void alloc_and_load_mux_graph_recursive(t_mux_node* node,
     node->starting_pin_idx = starting_pin_idx;
 
     if (level != 0) {
-        node->children = (t_mux_node*)vtr::calloc(node->num_inputs,
-                                                  sizeof(t_mux_node));
+        node->children = new t_mux_node[node->num_inputs];
+        for (int i = 0; i < node->num_inputs; i++)
```

Comment:
This for loop is unnecessary, as the default constructor will automatically be called on each element of the new-allocated array.